openfl/transport/grpc/aggregator_server.py

```diff
@@ -50,6 +52,7 @@ def __init__(
         root_certificate=None,
         certificate=None,
         private_key=None,
+        clients_certs_refresher_cb=None,
```

Comment:
Could you add a line in the docstring for the new parameter?

PS: shouldn't it be `client_cert_refresher_cb` - given that we fetch a single collaborator's cert?

Reply:
I've changed the name to `root_certificate_refresher_cb` as it is not suitable for reading a specific client certificate. Instead, it should read the root certificate which allows the aggregator to verify the client certificate that is given during the TLS handshake (indeed, in the case of self signed certificates, the root certificate is a chain of the clients certificates)